ocs_ci/ocs/ui/deployment_ui.py

```
@@ -485,13 +486,65 @@ def install_internal_cluster(self):
+                        By.XPATH,
+                    ),
+                    enable_screenshot=True,
+                )
+            elements = self.get_elements(
+                locator=self.dep_loc["enable_forceful_deployment"]
+            )
+            if elements:
+                raise ConfigurationError(
+                    "'Enable forceful deployment' checkbox should "
+                    "not appear in non-LSO deployment flow"
+                )
```

Ruff (0.15.21)

[warning] Abstract `raise` to an inner function

(TRY301)